packages/firestore/src/remote/bloom_filter.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Md5, Integer } from '@firebase/webchannel-wrapper';
+
+import { debugAssert } from '../util/assert';
+
+export class BloomFilter {
+  private readonly bitSize: number;
```

Comment:
nit: Rename `bitSize` to `numBits`, or even just `size` or `length`. IMO "bitSize" sounds like the "size of a bit" or the "number of bits in a byte" or something like that, which isn't what it is.